bin/docker_build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Get the directory to build argument
+if [ -n "$1" ] && [ "$1" != "--rebuild" ]; then
+  directory_to_build=$1
+elif [ -n "$2" ] && [ "$2" != "--rebuild" ]; then
+  directory_to_build=$2
+fi
```

Comment:
What are the two different styles of calling this script, and why are they both necessary?

Reply:
You can do one of the following: script, script --rebuild, script edudoc, script documents, script edudoc --rebuild, script documents --rebuild, script --rebuild edudoc, script --rebuild documents.

Reply:
Okay, in that case it should be fine as-is.